grpcclient/grpcclient.go

```diff
@@ -23,6 +23,15 @@ type Config struct {
 	GRPCCompression string  `yaml:"grpc_compression"`
 	RateLimit       float64 `yaml:"rate_limit"`
 	RateLimitBurst  int     `yaml:"rate_limit_burst"`
+	// KeepaliveTime is the number of seconds after which the client will ping the server in case of inactivity.
+	//
+	// See `google.golang.org/grpc/keepalive.ClientParameters.Time` for reference.
+	KeepaliveTime int64 `yaml:"ping_time"`
```

Comment:
Their CLI flags are not registered.

Reply:
@pracucci resolving this as these config params are converted into method arguments now.